docs/sources/architecture/distributor.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Distributor
+weight: 1000
+---
+# Distributor Component
+
+This document builds upon the information in the [Loki Architecture](./) page.
+
+## Where does it live?
+
+The distributor is the first component on Loki's write path downstream from any gateways providing auth or load balancing. It's responsible for validating, preprocessing, and applying a subset of rate limiting to incoming data before sending it to the ingester component. It is important that a load balancer sits in front of the distributor in order to properly balance traffic to them.
```

Comment:
> It is important that a load balancer sits in front of the distributor 

Question (doesn't have to in the doc, but just curious): in our setup say in dev or ops-cluster, what( and where) is the component? 
So when I send my logs via promtail to the client URL. That URL is basically the gateway and does Load balancing right?

Reply:
We have a bunch of extra stuff in front of this for Grafana Cloud, but it doesn't directly affect this open source project. Happy to talk about that in a less public place :).